### PR TITLE
docs: iconify GitHub link

### PR DESCRIPTION
**Pull request by juliusmarminge**
Closes #<issue>

## ✅ Checklist

- [ ] I have followed every step in the [contributing guide](https://github.com/t3-oss/create-t3-app/blob/main/CONTRIBUTING.md) (updated 2022-10-06).
- [ ] The PR title follows the convention we established [conventional-commit](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] I performed a functional test on my final commit

---

## Changelog

---

## Screenshots

_[Screenshots]_

💯

### CODE REVIEW

**Comment by juliusmarminge** [comment]
left a few comments on the vercel preview

**Comment by juliusmarminge** [comment]
Not perfect, feel free to adjust the styling

**Review by c-ehrlich — COMMENTED** [review]
Is this the same as what was merged in #648 ? If so, close this?

**Comment by bte234** [comment]
That PR was merged into this one, not next, so this one will still need to be merged
Maybe I should have it try to merge to next instead?

**Comment by juliusmarminge** [comment]
> That PR was merged into this one, not next, so this one will still need to be merged
> Maybe I should have had it try to merge to next instead?

No we merge this to next.

**Comment by bte234** [comment]
Cool, that's good to know
lmk if anything else is needed :D

**Comment by juliusmarminge** [comment]
> Cool, that's good to know
> lmk if anything else is needed :D

I think the transition states are a bit all over the place, some hovers transitions and some don't, and when switching themes some stuff transitions and some doesn't.

File: www/src/components/navigation/githubIcon.astro [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<style>
+  #github-star {
+    transition: all 300ms ease !important;
+  }
+  #github-link:hover {
+    text-decoration: none !important;
+  }
+  #github-icon {
+    transition: all 300ms ease !important;
+  }
```
Comment (juliusmarminge) [inline www/src/components/navigation/githubIcon.astro:51]: these 2 transitions won't take effect

Comment (juliusmarminge) [reply www/src/components/navigation/githubIcon.astro:51]: CleanShot 2022-10-24 at 17 24 51

**Comment by juliusmarminge** [comment]
merging as "good enough" - someone in disc will continue and making transitions more uniform etc

**Pull request merged**